### ISSUE TITLE
Sign ExternalMob with Citizen {Issue}

### ISSUE THREAD

**Issue by KevinBaselTheAnalyst** [DRE2N/DungeonsXL#232]
Hi Sataniel!
First of all thanks for you amazing job and fast answer thats you give to all of us!

Today I'm trying to create an ExternalMob Sign connecting with Citizens. I've tried different solutions, but seems that I've missed something because the NPC is not spawning like if the sign doesn't exist. This is what I've done:

1) Created an NPC into my main world and call him Polpo

2) Created a Sign into the Dungeons with this specification: 
Line 1 = ExternalMob
Line 2 = Polpo
Line 3 = 3,10,CI
Line 4 = D 15

What am I missing? Thanks in advance for you kindness!
Cheers

**Comment by Sataniel98** [comment]
Use the numeric ID for Citizens.

**Comment by KevinBaselTheAnalyst** [comment]
Everything is working now thanks! The NPC spawns, and I gave him some Sentinel command, like attack players,.... The problem now is that he attacks players and makes damage, but when other players attack him, they make no damage at all. Players are in gamemode 0 ofc, and I've set the proper targets with sentinel. I've tried to set the same NPC in the main world to see what happens and it works well. Seems I should change something inside DXL plugin. Sounds familiar?
P.S. Normal Mob Spawn Sign works well, players make damage to mobs, dunno why doesn't working with Citizens+Sentinel
